packages/mixin-cache-as-bitmap/src/index.ts

```diff
@@ -247,6 +282,15 @@ DisplayObject.prototype._initCachedDisplayObject = function _initCachedDisplayOb
         resolution: this.cacheAsBitmapResolution || renderer.resolution,
     });
 
+    if (this.cacheAsBitmapMultisample !== undefined)
+    {
+        renderTexture.framebuffer.multisample = this.cacheAsBitmapMultisample;
+    }
+    else if (renderer.gl.getContextAttributes().antialias)
+    {
+        renderTexture.framebuffer.multisample = MSAA_QUALITY.HIGH;
```

Comment:
Multisampling was introduced as an experimental feature. I'd be careful in making it default behavior just yet.

Reply:
Agreed